### ISSUE TITLE
Notebook corrupted when sending computer to sleep

### ISSUE THREAD

**Issue by BatteryModel** [microsoft/AzureNotebooks#173]
### Azure Notebooks environment 
Arrived at a notebook from Azure Notebooks (notebooks.azure.com)

### Description
My notebooks get corrupted when sending my laptop to sleep with notebook opened in tab.
When I try to access it again, all data is wiped and I get the following error message:
![bad request](https://user-images.githubusercontent.com/31781342/30347459-f6fd7c16-97c0-11e7-812e-976a7e952fda.gif)

And the content of the notebook becomes (via edit file option):
{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<script>\n",

I lost two different notebooks in a week. Thankfully I had downloaded backups the day before and was able to retrieve most of my work but this is an inexcusable flaw for a professional tool.

At the very least there should be an option to revert to the latest non-corrupted checkpoint.

### Repro Steps
Create a Jupyter notebook with a Python 3.6 kernel.
Make some edits.
Save.
Make some more edits.
Leave Notebook open in browser tab.
Send PC into sleep or hibernate.
Wake up PC.
Notebook is corrupted.

### Recommended Metadata
The Jupyter Kernel (Python2, Python3, R, N/A): Python 3.6
Occurrence Time: 
Web Browser: Opera
LibraryID and Notebook Name (if applicable): Private
Logs (!cat .nb.log Or: New/Terminal, $cat .nb.log) - attach files if applicable

**Comment by crwilcox** [comment]
How long are you leaving your machine hibernated or asleep? It is possible we are just disconnecting you due to inactivity. There is some oddness sometimes as their is client side javascript for jupyter that leaves you with a running front end. If you go back to notebooks.azure.com and relaunch your library, are the contents corrupted?

**Comment by seraph-zaho** [comment]
Hi, I have the same error. I tried to relaunch the library and it didn't work. I use Jupiter Azure Notebooks in my class and some os my students have the same mistake after uploading a ".ipynb" file that I shared with them. It's there any other solution for this issue?
![captura de pantalla 2017-09-26 a la s 10 17 28](https://user-images.githubusercontent.com/17915957/30869706-7cb05d74-a2a7-11e7-9735-37fad5406bea.png)
